src/styles/navlist.scss

```diff
@@ -0,0 +1,57 @@
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height fir elements with content inside. add padding to links instead